src/main/java/uk/gov/justice/digital/delius/service/OffenderService.java

```diff
@@ -181,8 +184,10 @@ private Custody findCustodyOrThrow(Event activeCustodialEvent) {
                 .orElseThrow(() -> new CustodyNotFoundException(activeCustodialEvent));
     }
 
+    @NationalUserOverride
```

Comment:
This will use "NationalUser" rather than "APIUser". That user should have read rights on all offenders, APIUser seems to be missing 8 offenders out of the 2 million, possible because of the organisations associated with APIUser